src/EventStore.Client.PersistentSubscriptions/EventStorePersistentSubscriptionsClient.Info.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using EventStore.Client.PersistentSubscriptions;
+using Grpc.Core;
+
+#nullable enable
+namespace EventStore.Client {
+	partial class EventStorePersistentSubscriptionsClient {
+		/// <summary>
+		/// Gets the status of a persistent subscription to $all
+		/// </summary>
+		/// <param name="groupName"></param>
+		/// <param name="deadline"></param>
+		/// <param name="userCredentials"></param>
+		/// <param name="cancellationToken"></param>
+		/// <returns></returns>
+		public async Task<PersistentSubscriptionInfo> GetInfoToAllAsync(string groupName, TimeSpan? deadline = null,
+			UserCredentials? userCredentials = null, CancellationToken cancellationToken = default) {
+
+			var channelInfo = await GetChannelInfo(cancellationToken).ConfigureAwait(false);
+			if (channelInfo.ServerCapabilities.SupportsPersistentSubscriptionsGetInfo) {
+				var req = new GetInfoReq() {
+					Options = new GetInfoReq.Types.Options{
+						GroupName = groupName,
+						All = new Empty()
+					}
+				};
+
+				return await GetInfoGrpcAsync(req, deadline, userCredentials, channelInfo.CallInvoker, cancellationToken)
+					.ConfigureAwait(false);
+			}
+
+			throw new NotSupportedException("The server does not support persistent subscriptions to $all.");
```

Comment:
this message about supporting persistent subscriptions to $all doesn't match the server cap that we checked `SupportsPersistentSubscriptionsGetInfo` when there is one that matches (`SupportsPersistentSubscriptionsToAll`)  if its correct, it probably warrants a comment in explanation

Reply:
Should the error be `The server does not support getting persistent subscription details for $all`?